apps/src/applab/designModeHeaders.jsx

```diff
@@ -0,0 +1,17 @@
+var React = require('react')
+var applabMsg = require('./locale')
+
+module.exports = React.createClass({
+  render: function() {
+    return (
+      <div>
+        <div id="design-toolbox-header" className="workspace-header">
+          <span>{applabMsg.designToolboxHeader()}</span>
+        </div>
+        <div id="design-workspace-header" className="workspace-header">
+          <span>{applabMsg.designWorkspaceHeader()}</span>
+        </div>
+      </div>
+    )
+  }
+});
```

Comment:
Renamed these two files. Not touching designProperties.jsx since capitalizing it seems like it would break everyone's OS X git clients.